_sass/3-layout/_global.scss

```diff
@@ -0,0 +1,103 @@
+@import "./0-base/reset";
+@import "./1-modules/mixins";
+@import "./1-modules/colors";
+@import "./1-modules/selectBox";
+@import "./1-modules/stickyAside";
+
+@import "./list";
+@import "./email";
+
+body {
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -moz-font-feature-settings: "kern", "liga", "clig", "calt";
+  -ms-font-feature-settings: "kern", "liga", "clig", "calt";
+  -webkit-font-feature-settings: "kern", "liga", "clig", "calt";
+  font-feature-settings: "kern", "liga", "clig", "calt";
+  background-color: var(--bg);
+  font-family: 'Source Code Pro', 'Helvetica', 'Arial', sans-serif;
+  font-size: 16px;
+  line-height: 1.6;
+  font-weight: 400;
+  color: var(--text);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+  border-bottom: 1px solid var(--link_subtle);
+
+  &:hover {
+    border-color: var(--subtle_);
```

Comment:
I’d recommend revisiting this tbh, the pretty blue-gray changes from the previous PR are perhaps _too_ subtle now in context 😅

Reply:
I'm down with revisiting this but there isn't a styleguide mapped out for this correct? No figma board we can play with?